rewrite-java/src/test/java/org/openrewrite/java/LombokUtilityClassTest.java

```diff
@@ -0,0 +1,314 @@
+package org.openrewrite.java;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+/**
+ * Cases to test:
+ * - Interfaces
+ * - Empty classes & interfaces
+ * - inheritance
+ * - abstract classes
+ * - instantiations of changed classes
+ * - constructor
+ * - public static void main
+ */
+class LombokUtilityClassTest implements RewriteTest {
+
+    @Nested
+    class ShouldApplyLombokUtility implements RewriteTest {
+
+        @Test
+        void givenOneStaticMethod() {
+            rewriteRun(
+                    recipeSpec -> recipeSpec.recipe(new LombokUtilityClass()),
```

Comment:
You might want to override the `defaults` method so define the recipe to run once, instead of in each individual test.